jdat_notebooks/MRS_Mstar_analysis/JWST_Mstar_dataAnalysis_runpipeline.ipynb

```diff
@@ -8,8 +8,7 @@
     }
    },
    "source": [
-    "<a id=\"top\"></a>\n",
-    "# MIRI MRS Spectroscopy of a Late M Star 1 - Running the Pipeline"
+    "# MRS Pipeline"
```

Comment:
I would consider moving the "Source of Simulations" and "Pipeline Versions" categories after "Documentation" in order to limit confusion on the order across the repository. The same applies to the other two notebooks in this `MRS_Mstar_analysis` directory.